packages/pharos/src/components/input-group/pharos-input-group.ts

```diff
@@ -83,17 +84,13 @@ export class PharosInputGroup extends PharosTextInput {
 
   private _adjustPadding(event: FocusEvent) {
     const amount = event.type === 'focus' ? -1 : 1;
-    super['_input'].style.paddingLeft = `${
-      parseInt(
-        window.getComputedStyle(super['_input'], null).getPropertyValue('padding-left'),
-        10
-      ) + amount
+    this._input.style.paddingLeft = `${
```

Comment:
The invalidated input group text shifts a little on focus. Probably because it already has a 2px border without focus. Might need to adjust the logic on line 86.